test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -1030,6 +1078,32 @@ private static IEdmModel GetEdmModel()
             return edmModel;
         }
 
+
+        private static IEdmModel GetScaleAndSridModel()
+        {
+            string csdlTemplate = "<?xml version=\"1.0\" encoding=\"utf-8\"?>" +
+    "<edmx:Edmx Version=\"4.0\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\">" +
+    "<edmx:DataServices>" +
+    "<Schema Namespace=\"NS\" xmlns=\"http://docs.oasis-open.org/odata/ns/edm\">" +
+    "<ComplexType Name=\"Complex\">" +
+    "<Property Name=\"GeographyPoint\" Type=\"Edm.GeographyPoint\" SRID=\"variable\" />" +
+    "<Property Name=\"DecimalProperty\" Type=\"Edm.Decimal\" Nullable=\"false\" Scale=\"variable\" />" +
+    "</ComplexType>";
+
+            csdlTemplate += "</Schema>" +
+                "</edmx:DataServices>" +
+                "</edmx:Edmx>";

```

Comment:
Not sure why this can't be:
```suggestion
            string csdlTemplate = "<?xml version=\"1.0\" encoding=\"utf-8\"?>" +
    "<edmx:Edmx Version=\"4.0\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\">" +
    "<edmx:DataServices>" +
    "<Schema Namespace=\"NS\" xmlns=\"http://docs.oasis-open.org/odata/ns/edm\">" +
    "<ComplexType Name=\"Complex\">" +
    "<Property Name=\"GeographyPoint\" Type=\"Edm.GeographyPoint\" SRID=\"variable\" />" +
    "<Property Name=\"DecimalProperty\" Type=\"Edm.Decimal\" Nullable=\"false\" Scale=\"variable\" />" +
    "</ComplexType>" +
    "</Schema>" +
    "</edmx:DataServices>" +
    "</edmx:Edmx>";
```

Or is there a reason why part of the CSDL is assigned to `csdlTemplate` and the other concatenated after?